packages/terra-spacer/tests/nightwatch/spacer-spec.js

```diff
@@ -0,0 +1,319 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+/* eslint-disable no-unused-expressions */
+// Background color added for context. The orange background shows margin, while the green background shows padding.
+const resizeTo = require('terra-toolkit/lib/nightwatch/responsive-helpers').resizeTo;
+
+module.exports = resizeTo(['tiny', 'small', 'medium', 'large', 'huge', 'enormous'], {
+  // Just the spacer component with <div> tag as child and without any props set
+  'Displays a default spacer': (browser) => {
+    browser
+      .url(`${browser.launchUrl}/#/tests/spacer-tests/default`)
+      .expect.element('#spacer').to.be.present;
+  },
+
+  /*
+    Margin Applied to visual example 1
+    Padding Applied to visual example 2
+    Margin and Padding Applied to visual example 3
+    Margin, Padding and isInline Applied to visual example 4
+   */
+  'Displays spacer with small-2 margin': (browser) => {
+    browser
+      .url(`${browser.launchUrl}/#/tests/spacer-tests/SpacerSmall2`)
+      .expect.element('#spacerMarginSmall2').to.be.present;
+    browser.expect.element('#spacerPaddingSmall2').to.be.present;
```

Comment:
nit:  could add a .to.equal(true) to all of the expects on if the element is to be present or has css.
this would allow you to not have to turn off the eslint rule of no-unused-expressions above

Reply:
I'm fine leaving this as is.